ga4gh/datamodel/reads.py

```diff
@@ -251,6 +255,22 @@ def getSamFilePath(self):
         """
         return self._samFilePath
 
+    def getPrograms(self):
```

Comment:
This should be documented --- see the comments in #589 for the recommended approach (i.e., define and document in the superclass).